### DISCUSSION TITLE
Phone view issues

### DISCUSSION THREAD

**Discussion by aracelyca** [impress-org/givewp#7236] [Developer Q&A]
Hello,

The field width and font size do not properly fit 6 or 7-digit entries. The phone view of the goal is 50,000 and it does not fit on the box. How can I solve this issue?

What custom CSS would be needed to increase the field width and/or decrease the font size?

Hope you can help, thanks!

**Comment by jonwaldstein** [comment]
Hey @aracelyca, 

Thanks for reaching out!

In order to clarify the context, can you please send a screenshot of this issue?

Thanks
